src/test/compile-fail/E0504.rs

```diff
@@ -8,18 +8,22 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// revisions: ast mir
+//[mir]compile-flags: -Z borrowck=mir
+
 struct FancyNum {
     num: u8,
 }
 
 fn main() {
-    let fancy_num = FancyNum { num: 5 };
+    let mut fancy_num = FancyNum { num: 5 };
     let fancy_ref = &fancy_num;
 
     let x = move || {
```

Comment:
Could we add a secondary span label pointing at `move ||` saying "moved into this closure"?